src/controller/user.js

```diff
@@ -0,0 +1,42 @@
+const { getAllUsersService, createUserService } = require('../service/user')
+const { checkExistingUserOrEmail } = require('../dao/user')
+const bcrypt = require('bcrypt');
+
+const getAllUsers = async (req, res) => {
+  try {
+    const users = await getAllUsersService()
+    res.status(200).json(users)
+  } catch (error) {
+    res.status(500).json(error)
+  }
+}
+
+
+const createUser = async (req, res) => {
+  try {
+    const user = req.body.user
+    const FirstPassword = req.body.password
+    const email = req.body.email
+    const check = await checkExistingUserOrEmail(user, email)
+    if (check == false) {
```

Comment:
use === equals for comparison, please read about the difference between 2 an 3 equals
also there is a more concise way to do this.
you don't need to use await because the internal function has already implemented its own async await

`if (checkExistingUserOrEmail(user, email)) {
  // Rest of the code...
} else {
  // Rest of the code...
}`